Makefile

```diff
@@ -28,6 +28,12 @@ coqutil:
 clean_coqutil:
 	$(MAKE) -C $(DEPS_DIR)/coqutil clean
 
+kami:
+	$(MAKE) -C $(DEPS_DIR)/kami
+
+clean_kami:
+	$(MAKE) -C $(DEPS_DIR)/kami clean
+
```

Comment:
Detail: clean_kami should be called by clean too

Reply:
Ah I forgot that; just pushed the change, thanks!